trackpy/tests/test_feature.py

```diff
@@ -464,43 +413,35 @@ def test_rg(self):
         # http://www.physics.emory.edu/~weeks/idl/radius.html
 
         self.check_skip()
-        L = 101 
+        L = 101
         dims = (L, L + 2)  # avoid square images in tests
         pos = [50, 55]
-        cols = ['y', 'x']
-
-        SIZE = 2
-        image = np.ones(dims, dtype='uint8')
-        draw_gaussian_spot(image, pos, SIZE)
-        actual = tp.locate(image, 7, 1, preprocess=False,
-                           engine=self.engine)['size']
-        expected = SIZE
-        assert_allclose(actual, expected, rtol=0.1)
 
         SIZE = 3
         image = np.ones(dims, dtype='uint8')
-        draw_gaussian_spot(image, pos, SIZE)
-        actual = tp.locate(image, 11, 1, preprocess=False,
+        draw_feature(image, pos, (SIZE*2 + 1) * 3, rg=0.333)
```

Comment:
It would really help to explain here that `rg=0.333` is dimensionless, and that its value in pixels is meant to be `SIZE`.